gooddata-sdk/gooddata_sdk/compute/model/execution.py

```diff
@@ -3,25 +3,52 @@
 
 from typing import Any, Optional, Union
 
+from attrs import Factory, define, field
+
 import gooddata_afm_client.apis as apis
 import gooddata_afm_client.models as models
 from gooddata_sdk.compute.model.attribute import Attribute
 from gooddata_sdk.compute.model.filter import Filter
 from gooddata_sdk.compute.model.metric import Metric
 
 
+@define
+class TotalDimension:
+    idx: int
+    """index of dimension in which to calculate the total"""
+
+    items: list[str] = field(default=Factory(list))
```

Comment:
items: list[str] = field(factory=list)

Get rid of Factory import.

Reply:
fixed in last commit